generate/g_appcode.go

```diff
@@ -255,6 +256,9 @@ func (tag *OrmTag) String() string {
 	if len(ormOptions) == 0 {
 		return ""
 	}
+	if tag.Comment != "" {
+		return fmt.Sprintf("`orm:\"%s\" description:\"%s\"`", strings.Join(ormOptions, ";"), tag.Comment)
```

Comment:
`strings.Join(ormOptions, ";")` use `;` as the sperator. but you are using hard code here and alsp use space. please read the code first.